sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -1928,4 +1928,17 @@ class HiveDDLSuite
       }
     }
   }
+
+  test("SPARK-20680: Spark-sql do not support for void column datatype of view") {
+    withTable("t", "tabNullType") {
+      val client = spark.sharedState.externalCatalog.asInstanceOf[HiveExternalCatalog].client
+      client.runSqlHive("CREATE TABLE t (t1 int)")
+      client.runSqlHive("INSERT INTO t VALUES (3)")
+      client.runSqlHive("CREATE TABLE tabNullType AS SELECT NULL AS col FROM t")
```

Comment:
IIRC, hive 2 does't support this. Let's test with `CREATE VIEW AS ...` to be safer